docs/source/developer_zone/build_locally.rst

```diff
@@ -323,8 +336,21 @@ or with ``libmamba``:
 Tests
 *****
 
-You should be able to run the Python-based test suite:
+In order to run the Python-based test suite, you need to set the following environment variables:
+
+.. code::
+
+    export TEST_MAMBA_EXE=build/micromamba/micromamba
+    export MAMBA_ROOT_PREFIX=YOUR_MICROMAMBA_ROOT_PREFIX
+
+Then, from the ``mamba`` root directory, you should be able to run the tests:
 
 .. code::
+    cd ..
+    pytest micromamba/tests/
 
-    pytest ./micromamba/tests/
+Since running all the tests would take a great amount of time, you could choose to run only a specific test.
+To launch ``test_env`` for example, you can run:
```

Comment:
We can also mention the `-k` option to filter by test name (across file, or for a subset of a file).